servicetest/capabilities/test_caps.py

```diff
@@ -230,3 +230,17 @@ def test_defaults_are_applied(self):
             assert my_env_var == "TEST_ENV_VALUE"
         finally:
             sc.terminate()
+
+    def test_set_same_cap_twice(self):
+        """ Test that setting the same capability doesn't result in crashes
```

Comment:
Alright. I will update it to be more specific. Just as the comment for the error I expect.